assets/js/domain-verifier-checker.js

```diff
@@ -1,4 +1,4 @@
-//Browserify was used to create domain-verifier-bundle.js so that we can use 'require' in the browser
+//Webpack was used to create domain-verifier-bundle.js so that we can use 'require' in the browser
```

Comment:
Hey @mDuo13, sorry for the confusion. The comment was supposed to say Webpack instead of Browserify and the README is correct.